packages/yourloops/pages/hcp/team-members-table.tsx

```diff
@@ -342,97 +336,63 @@ function TeamMembersCards(props: TeamMembersProps): JSX.Element {
   const members = teamHook.getMedicalMembers(team);
   members.sort(sortTeamMembers);
 
-  const papers = members.map((member: TeamMember): JSX.Element | null => {
+  const papers = members.map((member: TeamMember): JSX.Element => {
     const userId = member.user.userid;
     const email = member.user.username;
 
     // TODO onSwitchAdminRole
 
-    let removeMemberButton: JSX.Element;
-    if (userIsAdmin && userId !== currentUserId) {
-      const handleClickRemoveMember = async (): Promise<void> => {
-        await onShowRemoveTeamMemberDialog(member);
-      };
-      removeMemberButton = (
-        <IconActionButton
-          icon={<PersonRemoveIcon />}
-          id={`team-members-list-${team.id}-paper-${email}-action-remove`}
-          onClick={handleClickRemoveMember}
-          className={classes.paperMemberRemove}
-        />
-      );
-    } else {
-      removeMemberButton = <div id={`team-members-list-${team.id}-paper-${email}-action-remove-empty`} />;
-    }
+    const handleClickRemoveMember = async (): Promise<void> => {
```

Comment:
same for refacto in this file, is it e-cps icon related ?

Reply:
The component didn't use conditionnal rendering, code was duplicated based on the Invitation Status. So I add the certified icon in this component and introduce conditionnal rendering